.github/workflows/test.yaml

```diff
@@ -38,10 +39,69 @@ jobs:
             reports/jest/junit.xml
             reports/junit/*.xml
           comment_mode: off
-      - name: Send Slack notification with job status.
-        if: always()
+  save-cache:
+    name: Save Cache
+    runs-on: ubuntu-20.04
+    permissions: {}
+    steps:
+      - name: Check out repository.
+        uses: actions/checkout@v3.0.2
+      - name: Cache Docker images.
+        uses: ./
+        with:
+          key: docker-cache-test
+      - name: Build an empty test Docker image.
+        run: |
+          docker build --tag empty - <<<'FROM scratch
+            LABEL description="empty image"'
+  restore-cache:
+    name: Restore Cache
+    needs:
+      - save-cache
+    runs-on: ubuntu-20.04
+    permissions: {}
+    steps:
+      - name: Check out repository.
+        uses: actions/checkout@v3.0.2
+      - name: Load test cache.
+        uses: ./
+        with:
+          key: docker-cache-test
+      - name: Verify Docker loaded empty image from cache.
+        run: |
+          description="$(docker inspect --format '{{ index .Config.Labels "description" }}' empty)"
+          [[ $description == "empty image" ]]
+  delete-cache:
+    name: Delete Cache
+    if: needs.save-cache.result == 'success'
+    needs:
+      - save-cache
+      - restore-cache
+    runs-on: ubuntu-20.04
+    permissions:
+      actions: write # for deleting cache
+    steps:
+      - name: Delete existing test cache.
+        run: >
+          curl
+          --request DELETE
+          --header "Accept: application/vnd.github.v3+json"
+          --header "Authorization: token $GITHUB_TOKEN"
+          "https://api.github.com/repos/ScribeMD/docker-cache/actions/caches?key=docker-cache-test&ref=$GITHUB_REF"
+  notify:
+    if: always()
+    needs:
+      - pre-commit
+      - save-cache
+      - restore-cache
+      - delete-cache
+    runs-on: ubuntu-20.04
+    permissions: {}
+    steps:
+      - name: Send Slack notification with workflow result.
         uses: ScribeMD/slack-templates@0.6.2
         with:
-          bot-token: ${{ secrets.SLACK_BOT_TOKEN }}
-          channel-id: ${{ secrets.SLACK_ACTIONS_CHANNEL_ID }}
+          bot-token: ${{ secrets.SLACK_TEMPLATES_BOT_TOKEN }}
+          channel-id: ${{ secrets.SLACK_TEMPLATES_CHANNEL_ID }}
```

Comment:
Please revert since these secret names come from the slack-templates documentation and are for explanatory purposes only.